bin/version

```diff
@@ -0,0 +1,3 @@
+#!/bin/env bash
+
+grep '(defproject clojupyter' project.clj | sed 's/.\+"\(.\+\)"/\1/g'
```

Comment:
I ran this on my machine which led to the output:

```
(defproject clojupyter "0.4.0"
```

while I'm no shell scripting wiz, I was able to make this output `0.4.0` using

```suggestion
grep '(defproject clojupyter' project.clj | sed 's/(defproject clojupyter//' | sed 's/.\+"\(.\+\)"/\1/g' | sed 's/"//g'
```
but it can probably be made more elegantly

Reply:
Both versions output `0.4.0` on my machine. I think our versions of sed differ slightly.

I think a simpler version would be to just skip the middle part, like so:

`grep '(defproject clojupyter' project.clj | sed 's/(defproject clojupyter //' | sed 's/"//g'`

Does the above code output the right thing on your machine?

Reply:
Also, does this command work as expected on your machine?

`grep '(defproject clojupyter' project.clj | sed -E 's/.+"(.+)"/\1/g'`

Reply:
👍🏻 `grep '(defproject clojupyter' project.clj | sed -E 's/.+"(.+)"/\1/g'`works on macOS too - so that's clearly better.